components/nimbus/ios/Nimbus/NimbusBuilder.swift

```diff
@@ -30,7 +30,7 @@ public class NimbusBuilder {
      * A closure for reporting errors from Rust.
      */
     @discardableResult
-    func withErrorReporter(_ reporter: @escaping NimbusErrorReporter) -> NimbusBuilder {
+    func with(errorReporter reporter: @escaping NimbusErrorReporter) -> NimbusBuilder {
```

Comment:
I'm not familiar with this code format, could you briefly explain what's happening here?

Reply:
Sure. Swift has named arguments _as part of the name of the method_, and the Swift trend has been to embrace that, so now we write:

```swift
builder.with(errorReporter: theReporter)
```

instead of:

```swift
builder.withErrorReporter(theReporter)
```

The syntax above is:

```swift
func methodName(namedArg localName: Type)
```

where `namedArg` is `_`, then it's omitted when calling (like `withErrorReporter(theReporter)`).
and `localName` is optional, and that's the name of the argument in the method.

```swift
func methodName(namedArg localName: Type) {

}
```

is syntactic sugar for:

```swift
func methodName(namedArg: Type) {
   let localName = namedArg

}
```

Let me know if this does/doesn't help.